### ISSUE TITLE
The pre-trained segmentation model for the evaluation for horse2zebra

### ISSUE THREAD

**Issue by CR-Gjx** [taesungp/contrastive-unpaired-translation#20]
Hi, thanks for your great and motivating work. I notice that this paper propose a new evaluation metric (detected pixels) in the horse2zebra dataset, but I cannot find the pre-trained segmentation model in this repository, can you provide it? 
Thanks!

**Comment by CR-Gjx** [comment]
I have found it in the Appendix. Thanks!

**Comment by taesungp** [comment]
Great! Yes we used the pretrained model at https://github.com/kazuto1011/deeplab-pytorch.
